README.md

```diff
@@ -58,6 +58,13 @@ tc.get_build_by_build_id(73450)
 tc.get_server_info()
 tc.get_agents()
 tc.get_all_plugins()
+tc.get_project_params('MyProjectId')
+```
+
+### Putting Data
```

Comment:
Can you add a little more information here instead of just the header `Putting Data` would be good to document exactly what you are showing.